curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-and-save-a-record-of-a-model.english.md

```diff
@@ -11,7 +11,7 @@ In this challenge you will have to create and save a record of a model.
 
 ## Instructions
 <section id='instructions'>
-Create a document instance using the <code>Person</code> constructor you built before. Pass to the constructor an object having the fields <code>name</code>, <code>age</code>, and <code>favoriteFoods</code>. Their types must be conformant to the ones in the Person Schema. Then call the method <code>document.save()</code> on the returned document instance. Pass to it a callback using the Node convention. This is a common pattern, all the following CRUD methods take a callback function like this as the last argument.
```

Comment:
Conformant is an adjective which seems OK in this sentence.  Conformed seems odd to me.  How about the following?
```suggestion
Create a document instance using the <code>Person</code> constructor you built before. Pass to the constructor an object having the fields <code>name</code>, <code>age</code>, and <code>favoriteFoods</code>. Their types must conform to the ones in the Person Schema. Then call the method <code>document.save()</code> on the returned document instance. Pass to it a callback using the Node convention. This is a common pattern, all the following CRUD methods take a callback function like this as the last argument.
```

Reply:
Yes, "Their types must conform" is easier to understand, especially for the second language people like me.

Reply:
Yes, "Their types must conform" is easier to understand, especially for people who English is the second language like me.